docker/docker/Dockerfile.linux.amd64

```diff
@@ -1,5 +1,5 @@
-FROM gcr.io/kaniko-project/executor:v1.9.1
+FROM gcr.io/kaniko-project/executor:v1.9.2
```

Comment:
Saw a thread recently that users get error with 1.9.2 while pushing large images. Can we once validate ?
https://github.com/GoogleContainerTools/kaniko/issues/2463